src/pages/offers/offer-feautures.tsx

```diff
@@ -0,0 +1,16 @@
+import classnames from 'classnames';
+import {Feauture} from '../../types';
+
+export default function OfferFeautures ({feauture, id}: Feauture): JSX.Element {
```

Comment:
И в других компонентах аналогично. Компонент получает на вход объект `props`, в котором уже внутри что-то мы задаём. И поэтому сам объект `props` не может быть `Feature`, а только может содержать в себе внутри объект с таким типом. Поэтому `props = OfferFeaturesProps`, внутри которого может быть `feature = Feature`

```suggestion
export default function OfferFeautures ({feauture, id}: OfferFeaturesProps): JSX.Element {
```